inttests/deploy_test.go

```diff
@@ -30,20 +31,20 @@ func TestDeployGetPackgeDetails(t *testing.T) {
 
 // TestDeployValidateMDMDetails function to test Retrival of MDM Topology Function
 func TestDeployValidateMDMDetails(t *testing.T) {
-	mapData := map[string]interface{}{
+	clusterData := map[string]interface{}{
 		"mdmUser":     "admin",
```

Comment:
Will the "mdmUser" always be admin? Is it possible to have any other user, if so please make it dynamic by fetching from new GOSCALEIO_MDMUSER key.

Reply:
For initial Deployment user will admin only. After deployment only we can add users. That's why used admin.